### ISSUE TITLE
Release 3.4.0 broken with Carthage

### ISSUE THREAD

**Issue by sebskuse** [checkout/frames-ios#57]
**Describe the bug**

As of the latest release we are unable to build the SDK with `Carthage`:

```
[16:55:37]: ▸ Dependency "frames-ios" has no shared framework schemes for any of the platforms: iOS
[16:55:37]: ▸ If you believe this to be an error, please file an issue with the maintainers at https://github.com/checkout/frames-ios/issues/new
```

It appears that the workspace has moved from the root directory, which means that Carthage is unable to find it.

**To Reproduce**
1. Add `frames-ios` as a carthage dependency
2. Attempt to build dependencies

**Expected behavior**
The project should build

**Comment by harry-brown-cko** [comment]
Hi @sebskuse,
Apologies for the Carthage issue. We are planning to release a patch for this next week. Please use version `3.3.0` until then.

**Comment by harry-brown-cko** [comment]
Hey @sebskuse, thanks for the heads up. I think we're almost there with the fix now in PR #58. We have tested ourselves and it should now be fine. If you want to double check, point your Cartfile at branch `bug/fix-carthage`.

**Comment by sebskuse** [comment]
Hi @harry-brown-cko - just given this a go and it looks like it compiles after replacing `FramesIos` with `Frames` in our codebase, thanks!

As an aside, as a feature request - it would be great to drop the `Alamofire` dependency in favour of just using `URLSession`. Nowadays `URLSession` is just as powerful, there's less chance of a conflict between other dependencies that may use a different locked version of AF, and it's one less dependency for apps to bundle.

**Comment by harry-brown-cko** [comment]
Yeah @sebskuse we agree. This is definitely something we're considering for future releases.

**Comment by harry-brown-cko** [comment]
@sebskuse this issue was fixed as part of the latest release, `3.4.1`. Please feel free to re-open this if you have any further questions.
